README.md

```diff
@@ -19,10 +19,49 @@ Using GraphQL with server-side rendering in React is a challenging problem. Curr
 
 ### Installation
 
+Install `next-urql` along with its `peerDependencies`.
+
 ```sh
-yarn add next-urql
+yarn add next-urql react-is styled-components isomorphic-unfetch
+```
+
+`react-is`, `styled-components`, and `isomorphic-unfetch` help to support server-side `Suspense` with `react-ssr-prepass`. This assumes you have followed the basic installation steps for `urql` [here](https://github.com/FormidableLabs/urql#installation).
```

Comment:
isn't the "," after `styled-components` redundant?

Reply:
❤️ [Oxford comma](https://www.grammarly.com/blog/what-is-the-oxford-comma-and-why-do-people-care-so-much-about-it/)

Reply:
Today I learned, thanks for that! resolving the comment